apps/landing/vite.config.ts

```
@@ -1,8 +1,13 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
+import tailwindcss from "@tailwindcss/vite";
+import { defineConfig } from "vite";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'defineConfig'. Consider to delete it or rename it.

'defineConfig' is defined here:


(lint/suspicious/noRedeclare)